### DISCUSSION TITLE
Passing extra query paramaters in OIDC?

### DISCUSSION THREAD

**Discussion by samilliken** [goharbor/harbor#17363] [Ideas]
Hello, we have a business need to utilize OIDC for multiple accounts for a single user - something our IdP is already configured to handle properly.   However, to do that without having to constantly open multiple Incognito Windows we would need to pass an extra query parameter back to AzureAD.   Specifically, prompt: "select_account".   Could we possibly have an extra field in the OIDC configuration page to allow for extra query data to be passed?

**Comment by Vad1mo** [comment]
To my knowledge, this should already be possible, look for `oidc_extra_redirect_parms`

- https://github.com/goharbor/harbor/blob/main/src/server/v2.0/models/configurations_response.go#L122-L122
- https://github.com/goharbor/harbor/issues/13092

**Reply by samilliken** [reply]
Aha, thanks for that!   I see that it was added as an API access only, which is fine for me, but having it as a field in the UI would certainly make it more accessible and obvious - especially if someone puts in something wrong and is trying to debug.
